ui/app/pages/routes/routes.component.js

```diff
@@ -101,12 +101,6 @@ export default class Routes extends Component {
     this.props.history.listen((locationObj, action) => {
       if (action === 'PUSH') {
         pageChanged(locationObj.pathname)
-        this.context.metricsEvent({}, {
```

Comment:
I'm not sure what this event was meant to track, but with our new page tracking in segment I don't believe it to be required. It is also the only matomo event that didn't have category and name.

Reply:
This was how we did page tracking with Matomo. The new Segment page tracking effectively replaces it.